packages/stt-adapters/digital-paper-edit/index.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Convert Digital Paper Edit transcript json format to DraftJS 
+ * More details see
+ * https://github.com/bbc/digital-paper-edit
+ */
+import generateEntitiesRanges from '../generate-entities-ranges/index.js';
+import groupWordsInParagraphsBySpeakers from './group-words-by-speakers.js';
+/**
+ * groups words list from kaldi transcript based on punctuation.
+ * @todo To be more accurate, should introduce an honorifics library to do the splitting of the words.
+ * @param {array} words - array of words opbjects from kaldi transcript
+ */
+const groupWordsInParagraphs = words => {
+  const results = [];
+  let paragraph = { words: [], text: [] };
+
+  words.forEach(word => {
+    // if word contains punctuation
+    if (/[.?!]/.test(word.text)) {
+      paragraph.words.push(word);
+      paragraph.text.push(word.text);
+      paragraph.text = paragraph.text.join(' ');
+      results.push(paragraph);
+      // reset paragraph
+      paragraph = { words: [], text: [] };
+    } else {
+      paragraph.words.push(word);
+      paragraph.text.push(word.text);
+    }
+  });
+
+  return results;
+};
+
+const digitalPaperEditToDraft = (digitalPaperEditTranscriptJson) => {
+  const results = [];
+  let tmpWords;
+  let speakerSegmentation = null;
+  let wordsByParagraphs = [];
+
+  tmpWords = digitalPaperEditTranscriptJson.words;
+  if (digitalPaperEditTranscriptJson.paragraphs !== undefined) {
+    speakerSegmentation = digitalPaperEditTranscriptJson.paragraphs;
+  }
+
+  if (speakerSegmentation === null) {
```

Comment:
a more graceful way of doing this is just `if (!speakerSegmentation) {`

Reply:
awesome thanks, used that logic/pattern to adjust a few others things like that, and used the automatic test to make sure it's all still good :)